loqui/im
# Switch to libphonenumber.js

This potentially fixes many issues.

_Testing procedure_
Open the WhatsApp account on foreground, go to Contacts tab and run this on App Manager:

```
Messenger.account().core.roster = [];
Messenger.account().save();
Messenger.account().allRender();
```

Then tap the sync button and see if your contacts reappear and you can see their avatars and statuses.
